src/ProfileEndpoints.jl

```diff
@@ -315,6 +315,22 @@ function handle_task_backtraces(stage_path = nothing)
     return HTTP.Response(200, backtrace_file)
 end
 
+###
+### Profile removal
+###
+
+function handle_profile_removal(stage_path = nothing)
+    if stage_path !== nothing
+        # Remove all files in the stage path
+        for file in readdir(stage_path)
+            if isfile(joinpath(stage_path, file))
+                rm(joinpath(stage_path, file))
+            end
+        end
```

Comment:
Shouldn't we be checking, that we're only removing profiles?

Reply:
Add a check for the file extension.